BE/src/main/java/team29/hoorry/issuetracker/core/jwt/JwtValidator.java

```diff
@@ -0,0 +1,27 @@
+package team29.hoorry.issuetracker.core.jwt;
+
+import io.jsonwebtoken.JwtException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import team29.hoorry.issuetracker.core.exception.ExceptionMessage;
+import team29.hoorry.issuetracker.core.exception.InvalidTokenException;
+import team29.hoorry.issuetracker.core.member.MemberRepository;
+
+
+@Component
+@RequiredArgsConstructor
+public class JwtValidator {
+
+	private final MemberRepository memberRepository;
+
+	public boolean validate(String bearerAuthorization) {
+		try {
+			Long memberId = JwtParser.parseClaim(bearerAuthorization, "memberId", Long.class);
+			memberRepository.findById(memberId)
+				.orElseThrow(() -> new InvalidTokenException(ExceptionMessage.NO_MEMBER_MESSAGE));
```

Comment:
```suggestion
			Optional<Member> member = memberRepository.findById(memberId);
                        if (!member.isPresent()) {
                            throw new InvalidTokenException(ExceptionMessage.NO_MEMBER_MESSAGE));
                        }
```

현재 코드에서 멤버가 존재하지 않으면 예외를 발생시키기 위해서 리턴값 없이 orElseThrow를 사용하였는데,
suggestion처럼 하는 것과 기존 코드 중에 어느 것이 더 괜찮은 방법일까요?